### PR TITLE
[SPARKTA-274] Fault tolerant

### PR DESCRIPTION
**Pull request by anistal**
#### Description
With this PR Sparkta should create from a checkpoint path a SparkStreamingContext with the saved data.

### CODE REVIEW

**Comment by gserranojc** [comment]
+1 

Jira 2w jaja

**Comment by danielcsant** [comment]
We should delete the policy checkpoint when updating the policy

**Comment by danielcsant** [comment]
test-at ISocketOParquetOperatorsIT fails

File: serving-core/src/main/scala/com/stratio/sparkta/serving/core/constants/AppConstant.scala [hunk 1]
```diff
@@ -132,4 +133,7 @@ object AppConstant {
     s"Ingestion${Parser.ClassSuffix}" -> s"parser-ingestion$pluginExtension",
     s"Type${Parser.ClassSuffix}" -> s"parser-type$pluginExtension"
   )
+
+  def checkpointPath(policy: AggregationPoliciesModel): String =
```
Comment (danielcsant) [inline serving-core/src/main/scala/com/stratio/sparkta/serving/core/constants/AppConstant.scala]: I believe this is not the place to put this function. Could be in AggregationPoliciesModel?

File: driver/src/main/scala/com/stratio/sparkta/driver/service/StreamingContextService.scala [hunk 2]
```diff
@@ -47,8 +47,16 @@ case class StreamingContextService(policyStatusActor: Option[ActorRef] = None, g
 
   def standAloneStreamingContext(apConfig: AggregationPoliciesModel, files: Seq[File]): Option[StreamingContext] = {
     runStatusListener(apConfig.id.get, apConfig.name)
-    SparktaJob.runSparktaJob(getStandAloneSparkContext(apConfig, files), apConfig)
-    SparkContextFactory.sparkStreamingInstance
+
+    val ssc = StreamingContext.getOrCreate(s"${apConfig.checkpointPath}/${apConfig.name}", () => {
```
Comment (danielcsant) [inline driver/src/main/scala/com/stratio/sparkta/driver/service/StreamingContextService.scala]: Use here also the common method `checkpointPath` you have extracted out

**Pull request closed without merging**